### DISCUSSION TITLE
TIN/EIN numbers are meaningless unless they can be tied to facility names/locations?

### DISCUSSION THREAD

**Discussion by tylerrollinsphd** [CMSgov/price-transparency-guide#693] [Q&A]
As a consumer, if I look up procedure A#### and I see that for NPI 1234567 there is a negotiated rate of $50 for the TIN/EIN of 99-111111 and there is a negotiated rate for the same NPI at TIN.EIN 92-88888 for $35, how can I decide where to go if all there are is EIN numbers and no gov't database/crosswalk by which to find the associated hospital and address for the given TIN/EIN. IRS has a TIN matching service but you must have the TIN/EIN and the business name and it will return whether they match or not, which isn't particularly helpful for this use case. How are folks matching the business name to the TIN/EIN for this data? If they were required to use NPI2 I think that would fix make it easier but so far I am only seeing TIN/EIN data and no reliable way to make the connection to the business name and address. Any thoughts?

**Comment by Nabins** [comment]
There are few websites from which we can pull details of ein based on Employer Identification Number (EIN). Some are free but some are paid. one example of free website is:
https://eintaxid.com/
I can help if you have any confusions. Please let me know

**Reply by nfanous** [reply]
Could you link all the other sites you're aware of, including the paid ones?

**Comment by tylerrollinsphd** [comment]
This is a small sample of ein/business names. In a random sample it returns
less than 10% match. So if you have 100 EINs in the dataset only about 10
will be in.that database you linked.

**Comment by magician25** [comment]
You are right. This is a big issue making entire exercise of disclosing the EINs useless when we don't know the facility name. 

The [NPI final rule](https://www.cms.gov/regulations-and-guidance/administrative-simplification/nationalprovidentstand/downloads/npifinalrule.pdf) of 2004 specifically made EIN numbers FOIA disclosable for healthcare entities (NPI-2 types). However, many one-person incorporated entities filled their SSN (instead of EIN) while applying for NPI so CMS delayed the disclosure of EIN in the NPI dissemination file in 2007 and said they will disclose it later. It has been over 15 years now, however, CMS still hasn't made it public. 

I raised a FOIA with CMS for this data (NPI-EIN mapping for NPI-2 entities) in 2017 which they denied (funnily citing that EIN is private tax information) - I didn't pursue it further. I hope someone at CMS can take it up and finally disclose the EIN numbers for NPI-2 entities. That will resolve the issue of the mapping here as well. 

BTW, I am also the data engineer behind eintaxid.com quoted above and have been looking for this information. 

<img width="781" alt="Screenshot 2023-09-28 at 4 23 52 PM" src="https://github.com/CMSgov/price-transparency-guide/assets/6287010/3efa904e-fd95-43f2-8cc2-0f0a2e789521">

**Reply by tylerrollinsphd** [reply]
I requested a selection of tins/eins and HHS said they don't have that information but it's very clearly something they should have in NPPES. If you'd like to work together to get this info, reach out to me

**Reply by magician25** [reply]
You can start with an FoIA request. May be HHS/CMS change their stance now. Please let us know of the result

**Reply by Nabins** [reply]
> I requested a selection of tins/eins and HHS said they don't have that information but it's very clearly something they should have in NPPES. If you'd like to work together to get this info, reach out to me

I am interested to work together to get this. How can we connect?

**Reply by tjcouey** [reply]
I would also like to help address this. I’m the chairman of my state trade association and run a nonprofit to represent families who cannot access behavior healthcare they need, both which could be useful dealing with HHS

**Comment by tjcouey** [comment]
Any progress on this issue? We are developing a solution and are currently using the providers address derived from their NPI as a location reference but this expects that the address information is current - and it is well known that providers rarely update their NPI records when changing practice locations. The only way to tie these providers to more accurate locations is to associate them with the type 2 NPI's or the EIN and use those addresses.

**Reply by magician25** [reply]
Tying to EIN address will not be helpful. Many clinics belonging to same corporation will have the same EIN where the address on EIN will be corporate office. NPI-1 to NPI-2 mapping is reasonable approach here.

**Comment by tjcouey** [comment]
*** Security Notice *** This message was encrypted by Bloom Behavior Therapy for transport to your organization. Please ensure any replies are also encrypted as this message may contain HIPAA protected content.
___________________________________________________________________
I’m interested in participating in this effort. We are creating a proprietary process to do this – but I agree that it should be part of the transparency requirement – especially considering that we have to pay to access the EIN data. NPPES data is downloadable.



T.J. Couey
Director of Operations and Owner
Bloom Behavior Therapy LLC

Chairman, ABA Advocates

Schedule a meeting with me here: https://calendly.com/tj-couey

Ph. (270) 908-0461 X.1001 Fx. (270) 366-0780
6201 Benton Road, Paducah KY, 42003
www.bloombehavior.com<http://www.bloombehavior.com/>

This email and its attachments may contain privileged and confidential information and may contain protected health information (PHI) intended solely for the use of Bloom Behavior Therapy LLC and the recipient(s) named above. If you are not the recipient or the employee or agent responsible for delivering this message to the intended recipient, by reading this signature, you are now aware the review, dissemination, distribution, printing, or copying of this email message and any attachments is strictly prohibited. If you have received this transmission in error, please notify the sender immediately at 270-908-0461 and permanently delete this email and any attachments.

**Comment by tjcouey** [comment]
*** Security Notice *** This message was encrypted by Bloom Behavior Therapy for transport to your organization. Please ensure any replies are also encrypted as this message may contain HIPAA protected content.
___________________________________________________________________
FOIA would take far too long. Compelling them to provide the information through the intent of the legislation is the best path forward. Transparency was intended to provide consumers FREE access to pricing information to compare costs between providers so they can obtain the most reasonable pricing. If the information provided doesn’t provide access to WHO, the providers are without incurring additional costs (i.e., paying to lookup the EIN), then the legislation doesn’t meet the mandate.

We need to identify who in CMS can speak authoritatively to this issue and address it with them.

T.J. Couey
Director of Operations and Owner
Bloom Behavior Therapy LLC

Chairman, ABA Advocates

Schedule a meeting with me here: https://calendly.com/tj-couey

Ph. (270) 908-0461 X.1001 Fx. (270) 366-0780
6201 Benton Road, Paducah KY, 42003
www.bloombehavior.com<http://www.bloombehavior.com/>

This email and its attachments may contain privileged and confidential information and may contain protected health information (PHI) intended solely for the use of Bloom Behavior Therapy LLC and the recipient(s) named above. If you are not the recipient or the employee or agent responsible for delivering this message to the intended recipient, by reading this signature, you are now aware the review, dissemination, distribution, printing, or copying of this email message and any attachments is strictly prohibited. If you have received this transmission in error, please notify the sender immediately at 270-908-0461 and permanently delete this email and any attachments.

**Comment by tjcouey** [comment]
*** Security Notice *** This message was encrypted by Bloom Behavior Therapy for transport to your organization. Please ensure any replies are also encrypted as this message may contain HIPAA protected content.
___________________________________________________________________
NPI’s have the same issue if the payor contract is with the Type2 NPI and the organization is a national or regional association with multiple locations.

T.J. Couey
Director of Operations and Owner
Bloom Behavior Therapy LLC

Chairman, ABA Advocates

Schedule a meeting with me here: https://calendly.com/tj-couey

Ph. (270) 908-0461 X.1001 Fx. (270) 366-0780
6201 Benton Road, Paducah KY, 42003
www.bloombehavior.com<http://www.bloombehavior.com/>

This email and its attachments may contain privileged and confidential information and may contain protected health information (PHI) intended solely for the use of Bloom Behavior Therapy LLC and the recipient(s) named above. If you are not the recipient or the employee or agent responsible for delivering this message to the intended recipient, by reading this signature, you are now aware the review, dissemination, distribution, printing, or copying of this email message and any attachments is strictly prohibited. If you have received this transmission in error, please notify the sender immediately at 270-908-0461 and permanently delete this email and any attachments.

**Reply by magician25** [reply]
No. Each location has a separate NPI-2. Infact, independent department within the same address have separate NPIs…e.g. pharmacy, blood bank, Lab within the same hospital have different NPIs

**Comment by tjcouey** [comment]
*** Security Notice *** This message was encrypted by Bloom Behavior Therapy for transport to your organization. Please ensure any replies are also encrypted as this message may contain HIPAA protected content.
___________________________________________________________________
I can’t speak to facilities like Hospitals – but I can for small groups (like Mental Health practices). They can absolutely have one NPI (the LLC) and multiple practitioners under it who are at multiple locations. Regardless – using that data is problematic because it relies on the owner of that NPI to keep their information current. Using something like CAQH would be much more accurate as it requires frequent attestation of the data.

T.J. Couey
Director of Operations and Owner
Bloom Behavior Therapy LLC

Chairman, ABA Advocates

Schedule a meeting with me here: https://calendly.com/tj-couey

Ph. (270) 908-0461 X.1001 Fx. (270) 366-0780
6201 Benton Road, Paducah KY, 42003
www.bloombehavior.com<http://www.bloombehavior.com/>

This email and its attachments may contain privileged and confidential information and may contain protected health information (PHI) intended solely for the use of Bloom Behavior Therapy LLC and the recipient(s) named above. If you are not the recipient or the employee or agent responsible for delivering this message to the intended recipient, by reading this signature, you are now aware the review, dissemination, distribution, printing, or copying of this email message and any attachments is strictly prohibited. If you have received this transmission in error, please notify the sender immediately at 270-908-0461 and permanently delete this email and any attachments.

**Comment by tylerrollinsphd** [comment]
HHS denied my request because they said they don't have tin/ein
information.

**Comment by JR-CMS** [comment]
HHS does not have a database for matching TINs to business names and address. Information for non-profits can be searched for on the IRS' website: https://apps.irs.gov/app/eos/.

We continue to monitor discussions here and appreciate the feedback on improving the machine-readable file schemas and reporting requirements.

**Reply by magician25** [reply]
They do in NPPES - EIN is part of nppes record and is supposed to be disclosable in NPI registry. Some people wrongly filled their SSN for one person practice etc and hence CMS decided to hide it for limited time in 2007. 

They can disclose it now, if they want to - regulation already allows them to do so…

**Comment by dgolden** [comment]
Caveat: My involvement in this area dropped off about 8-9 months ago, so
apologies if any of this information is dated.

The EIN/TIN database is owned by the IRS. When price transparency analysis
was first enabled by release of the MRFs I submitted an FOIA request to the
IRS.  I was denied and did a followup speaking to the IRS staff. I was
informed that EIN/TIN is confidential information and cannot be released.

All available (free online) databases for EIN/TIN cross-reference are
inadequate in their coverage.

No doubt the health claim clearing houses have compiled an excellent
cross-reference database. Due to the Federal government's failure in an
antitrust suit, Change Healthcare will be the only comprehensive source. I
seriously doubt its owner UnitedHealthcare would be amenable to release of
this valuable data.

As this discussion is heading off-topic, any follow-up questions should be
directed to my personal email  dgolden -at- gmail -dot- com

**Comment by tylerrollinsphd** [comment]
What can be done to get the business names and addresses? A consumer can't
make an informed decision without knowing where a doctor practices and the
rate at one facility vs another
